docs/reference/aggregations/pipeline/bucket-sort-aggregation.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[search-aggregations-pipeline-bucket-sort-aggregation]]
+=== Bucket Sort Aggregation
+
+A parent pipeline aggregation which sorts the buckets of its parent multi-bucket aggregation.
+Zero or more sort fields may be specified together with the corresponding sort order.
+Each bucket may be sorted based on its `_key`, `_count` or its sub-aggregations.
+In addition, parameters `from` and `size` may be set in order to truncate the result buckets.
+
+NOTE: The `bucket_sort` aggregation, like all pipeline aggregations, is executed after all other sibling aggregations.
+This means the sorting only applies to whatever buckets are already returned from the parent aggregation. For example,
+if the parent aggregation is `terms` and its `size` is set to `10`, the `bucket_sort` will only sort over those 10
+returned term buckets.
+
+==== Syntax
+
+A `bucket_sort` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+    "bucket_sort": {
+        "sort": [
+            {"sort_field_1": {"order": "asc"},<1>
+            {"sort_field_2": {"order": "desc"},
+            "sort_field_3"
+        ],
+        "from": 1,
+        "size": 3
+    }
+}
+--------------------------------------------------
+// NOTCONSOLE
+<1> Here, `sort_field_1` is the bucket path to the variable to be used as the primary sort and its order
+is ascending.
+
+.`bucket_sort` Parameters
+|===
+|Parameter Name |Description |Required |Default Value
+|`sort` |The list of fields to sort on. See <<search-request-sort,`sort`>> for more details. |Optional |
```

Comment:
Perhaps a section of the docs (or just a note, somewhere) which mentions you can use bucket_sort without any sorts, as a way to limit the number of buckets?  It's kinda implied since the docs say `sort` is optional, but maybe someone will overlook that?  It's such a useful part of the feature I'd hate people to miss it :)

Reply:
Added an example